### DISCUSSION TITLE
No UI or monika

### DISCUSSION THREAD

**Discussion by Gamerboo0fficial** [Monika-After-Story/MonikaModDev#9967] [QA - tech support + general]
hi i just reinstalled MAS on my macbook and i imported my save files and now when i boot the game it boots but monika isnt there ,dialog box or any of the sidebar options they appear for a second but then dissapear i cant click on the calender and i dont know what to do here is my save file(s) 
](https://drive.google.com/file/d/1YPUEVCLEzSUNG6MfaloGqeOKrIwtrGaT/view?usp=share_link)
[
](https://imgur.com/a/bROrgmN)

**Comment by Booplicate** [comment]
Restore a backup.

**Reply by Gamerboo0fficial** [reply]
that is the only backup i have

**Reply by Booplicate** [reply]
We make 10 backups for you, why don't you have any?

**Comment by Gamerboo0fficial** [comment]
well i do but its on a lost thumb drive so rn i dont have any and thats the
only save if theres a problem with it it would suck to start back at 0 but
if i must

**Reply by Hosna1100** [reply]
maybe I can give you a back up that I have 393 affection points in it but........_wait! does it count as cheating? or something illegal?_
